UIDD/UIDD-Session1/UIDD-Session1.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>UI Design and Development Session 1</title>
+</head>
+<body>
+
+
+	<!--Header-->
+	<header>
+		<div id="headerId" class="header">
+			<img src="images/footer-logo.png">
+			<div id="headerIdOne" class="headerOne">
+				<a href="home.html">Home</a>
+				<a href="features.html">Features</a>
+				<a href="reviews.html">Reviews</a>
+				<a href="download">Download</a>
+			</div>
+		</div>
+	</header>
+
+
+
+	<!--Cover-->
+	<div id="coverId" class="cover">
+		<div id="coverIdLeft" class="coverLeft">
+			<h1>Get the most fun weather app</h1>
+			<input type="button" name="download" value="Download">
+			<input type="button" name="features" value="Features">
+		</div>
+		<div id="coverIdRight" class="coverRight">
+			<img src="images/phone1.png">
+		</div>
+	</div>
+
+	<div id="coverIdTwo" class="coverTwo">
+		<h2>Perfect Features</h2>
+		<p>Only necessary</p>
+	</div>
+
+	<div id="coverIdThree" class="coverThree">
```

Comment:
we should add id and class attributes in the elements as per the requirements. (for example if class is used for styling same classname can be used at multiple places.)